lib/LaTeXML/Core/Token.pm

```diff
@@ -290,7 +322,7 @@ sub revert {
 
 sub toString {
   my ($self) = @_;
-  return $$self[0]; }
+  return $$self[1] == CC_ARG ? ("#" . $$self[0]) : $$self[0]; }
```

Comment:
I hijacked the `toString` method to create the TeX-near serialization, but maybe I should have consistently used `revert` instead? Most of the cases where I needed this were using `getString`, but I thought I should let that method reliably return the actual raw value in `$$self[0]`. Generally having this new arg token makes matters tricky, since its string serialization is different from its actual integer value (extra `#`).

Reply:
revert certainly sound more correct.  We possibly should scan all these variants and pin down the exact proper usecases and then use them consistently?